test/fabricator-postgres.spec.ts

```diff
@@ -0,0 +1,74 @@
+import * as pg from 'pg';
+import { Fabricator } from '../src/fabricator';
+import { PostgresAdaptor } from '../src/postgres-adaptor';
+import { exec } from 'child_process';
+import { expect } from 'chai';
+
+let dbConfig = {
+  host: 'localhost',
+  port: 5432,
+  user: 'dev',
+  password: 'test',
+  database: 'fabricator_test'
+};
+
+const conn = new pg.Client(dbConfig);
+
+describe('PostgresAdaptor', () => {
+  before((done) => {
+    Fabricator.setAdaptor(new PostgresAdaptor({ conn: conn }));
+    Fabricator.clearTemplate();
+
+    Fabricator.template({
+      name: 'organization',
+      attr: {
+        name: 'Fabricator Inc'
+      }
+    });
+    Fabricator.template({
+      name: 'department',
+      attr: {
+        name: 'IT',
+        organizationId: () => Fabricator.fabricate('organization').then(o => o.id)
+      }
+    });
+    Fabricator.template({
+      name: 'user',
+      attr: {
+        firstName: 'Bob',
+        lastName: 'Smith',
+        username: (obj) => `${obj.firstName}.${obj.lastName}`,
+        departmentId: () => Fabricator.fabricate('department').then(d => d.id)
+      }
+    });
+
+    let cmd = `PGPASSWORD=${dbConfig.password} psql -U ${dbConfig.user} -h ${dbConfig.host} -p ${dbConfig.port} postgres < ./test/sql/create-tables.postgres.sql`;
```

Comment:
```suggestion
    let cmd = `PGPASSWORD=${dbConfig.password} psql -U ${dbConfig.user} -h ${dbConfig.host} -p ${dbConfig.port} ${dbConfig.database} < ./test/sql/create-tables.postgres.sql`;
```

Reply:
Hey Reynard, sorry for late reply. I think the database should stay as `postgres` because the test database might not yet be created since its created through the `create-tables.postgres.sql` script. 
[It seems ](https://stackoverflow.com/questions/2370525/default-database-named-postgres-on-postgresql-server)when you install postgres, it always creates a `postgres` database, which is why I chose that one since from my testing, the `psql` cli always requires that you specify a database, otherwise it throws an error.